models/staging/tpdm/base/base_tpdm__survey_responses.sql

```diff
@@ -0,0 +1,33 @@
+with survey_responses as (
+    {{ source_edfi3('survey_responses') }}
+),
+renamed as (
```

Comment:
could we add a newline between CTEs?
```suggestion
),

renamed as (
```